### PR TITLE
Update authorization.adoc

### PR DESCRIPTION
**Pull request by psvanstrom**
Added information about errors.

### CODE REVIEW

**Comment by freddd** [comment]
Should we mention the header stuff as well?

**Comment by psvanstrom** [comment]
The header stuff is mentioned in the doc in the second section, the first section covers errors from the authorization server and there the specs mandate using response body errors as far as I understand.

**Comment by minisu** [comment]
Just to clarify, I'm talking about this stuff: https://github.com/iZettle/api-documentation/pull/7/files#diff-8025ed730e9afdb1d227d37ccdbd3f50R117

Didn't we say that ACCESS_TOKEN_EXPIRED only goes in the headers?

**Comment by psvanstrom** [comment]
The body will still be there, but perhaps we shouldn't document it since it might change?

The ACCESS_TOKEN_EXPIRED in the header will only happen once all services has updated to the latest authz-client so I didn't want to document it in detail just yet. Don't know if we should though?

**Comment by minisu** [comment]
Now we're saying that _"the resource service will return an error in the following format"_ and _"Additionally, the HTTP header `WWW-Authenticate` will contain similar information"_. I'm interpreting this as "all services [Finance and Purchase] returns the error code both in body and header". If this is true, why not just say that the error info is in the header, since the fact that it's also in the body is merely a bonus/legacy thing?

**Comment by psvanstrom** [comment]
Valid point! Although the error code won't be in the header for the time being, not until purchase has bumped the authz-client and the latest Finance has been released. I will try to rephrase it!

**Comment by psvanstrom** [comment]
@minisu Check now :)

File: authorization.adoc [hunk 1]
```diff
@@ -96,6 +109,14 @@ Access tokens should be included as follows:
 
 This is described in detail in https://tools.ietf.org/html/rfc6750#section-2.1[RFC 6750, section 2.1].
 
+#### Errors
+When using an expired access token in an API request, the resource service will return any error with the token in the HTTP header `WWW-Authenticate`, similar to this:
```
Comment (minisu) [inline authorization.adoc]: Maybe mention HTTP 401?

Comment (minisu) [inline authorization.adoc]: Looks good otherwise!

**Comment by psvanstrom** [comment]
@minisu Fixed!

**Comment by minisu** [comment]
Neat!

**Pull request merged**